packages/types/enums/user-status-enum.ts

```diff
@@ -0,0 +1,6 @@
+export const enum UserStatusEnum {
+  INACTIVE = 0,
+  CREATED = 1,
+  APPROVED = 2,
+  REPROVED = 3,
+}
```

Comment:
sugestão: recomendo muito também evitar de usar esses números mágicos para enumerações em banco de dados. Armazenar a string "inactive" ou "created" é mais caro sim, mas quase que praticamente insignificante. Se em algum momento for necessário consultar dados diretos no banco de dados e/ou alimentar eles em outro sistema, ter uma string direta vai fazer muito mais fácil de entender o que está acontecendo (evita de ter que ficar consultando "o que status 2 significa?")

Reply:
Realmente é muito mais legível a nível de banco de dados, irei padronizar para ficar no formato string